phastSim/phastSim.py

```diff
@@ -607,14 +693,96 @@ def normalize(node, norm):
                 node.rate /= norm
                 if node.isTerminal:
                     node.rates /= norm
+
                 else:
                     normalize(node.belowNodes[0], norm)
                     normalize(node.belowNodes[1], norm)
 
         # normalize the rates
         normalize(node=self.genomeRoot, norm=norm)
 
+        if self.indels:
+            # if there are indels then some further calculation is require to convert the rates from 
+            # substitution rates to overall rates.
+            
+
+            def include_indels(node, scale):
```

Comment:
I'm confused, why is this function defined both here and in line 773?

Reply:
I think this bit is commented out at the moment - I should have just deleted it. The reason why there are two is that I am trying to reuse the same function when creating the indels, but this won't quite work because the original function also changed self.norm = norm